src/malli/transform.cljc

```diff
@@ -69,7 +69,7 @@
 (defn -string->double [x]
   (if (string? x)
     (try #?(:clj  (Double/parseDouble x)
-            :cljs (let [x' (js/parseFloat x)] (if (js/isNaN x') x x')))
+            :cljs (let [x' (js/Number x)] (if (js/isNaN x') x x')))
```

Comment:
or should we use `cljs.core/parse-double` & `clojure.core/parse-double`?

Reply:
added a commit that switches to parse-double. note that parse-double was introduced recently in clojure(script) 1.11